css-typed-om/Overview.bs

```diff
@@ -784,10 +782,20 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |this|.
 </div>
 
-<div class=issue>
-    Define {{CSSNumericValue/equals()}}.
+<div algorithm="CSSNumericValue.equals(...|value|)">
+    The <dfn for=CSSNumericValue method>equals(...|value|)</dfn> method,
+    when called on a {{CSSNumericValue}} |this|,
+    must perform the following steps:
```

Comment:
The algo claims to be defined for CSSNumericValue, but the steps assume that you're looking at a CSSUnitValue.  It needs to work for all the CSSMathValue subtypes too.